README.md

```diff
@@ -63,6 +63,8 @@ For Linux users, run the `scripts/linux_preinstall.bash` script.
 
 > todo: finish MacOS preinstall script
 
+[Download `chromedriver`](https://sites.google.com/a/chromium.org/chromedriver/downloads) for your version of Google Chrome and move it to `/usr/bin/`.
```

Comment:
please follow 79 character line limit from MD style guide: https://github.com/voxel51/eta/blob/develop/docs/markdown_style_guide.md

Reply:
Ahhh, need to update vim settings on this machine